scripts/DocParser.js

```diff
@@ -124,6 +125,47 @@ function validate (docs, name, componentDirectory, strict) {
 	}
 }
 
+function parseTableOfContents (frontMatter, body) {
```

Comment:
I was going to suggest moving this to the gatsby node startup part, but it seems like it would make more sense to do this statically once here.  :+1: